### PR TITLE
Fix class hierarchy of ast nodes

### PR DESCRIPTION
**Pull request by serge-sans-paille**
Use expr/stmt inheritance.

### CODE REVIEW

File: gast/gast.py [hunk 1]
```diff
@@ -21,152 +21,207 @@ def create_node(self, *args, **kwargs):
...
-    'Del': ((), ()),
-    'AugLoad': ((), ()),
-    'AugStore': ((), ()),
-    'Param': ((), ()),
+    'Load': ((), (), (AST,)),
+    'Store': ((), (), (AST,)),
+    'Del': ((), (), (AST,)),
+    'AugLoad': ((), (), (AST,)),
+    'AugStore': ((), (), (AST,)),
+    'Param': ((), (), (AST,)),
```
Comment (pbrunet) [inline gast/gast.py]: You can go futher and provde expr_context here.

**Comment by pbrunet** [comment]
BTW, did you look at commonast project?

**Comment by serge-sans-paille** [comment]
On Tue, Feb 07, 2017 at 11:09:26AM -0800, pbrunet wrote:
> BTW, did you look at commonast project?

No! That's interesting, but a less cool name :-)

**Comment by serge-sans-paille** [comment]
ok!

**Pull request merged**